.github/workflows/test.yml

```diff
@@ -17,6 +18,10 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      - name: Run BASIC tests
```

Comment:
✅ Great addition! Running BASIC tests will help ensure code quality. Consider specifying what 'BASIC' tests entail for better understanding.